### DISCUSSION TITLE
メールでのアンケート依頼

### DISCUSSION THREAD

**Discussion by github895439** [increments/qiita-discussions#916] [要望・アイデア・不具合]
タイトルのものが送られて来ました。
送信元アドレスやアンケートページのリンクに怪しさはありませんでしたが、確証はありません。
アンケート内ではIDを問われていたため、偽物なら漏洩することになります。
メールはWebと機械的に紐づいていないためリスクがあります。
アンケートに限らず、QiitaのWebから依頼ページに行けるようにしてはどうでしょうか。

**Comment by tanaka-asami415** [comment]
@github895439 
ご意見ありがとうございます。不安にさせてしまい申し訳ありません。
取り急ぎ、トップページの右端のアナウンス欄に表示するようにいたしました🙇

![image](https://github.com/user-attachments/assets/8429891b-11fb-48af-9ffb-2448fa7cde50)
